test/functional/android/touch_action_tests.py

```diff
@@ -175,13 +175,13 @@ def test_drag_and_drop(self):
         el = wait_for_element(self.driver, MobileBy.ACCESSIBILITY_ID, 'Drag and Drop')
         action.tap(el).perform()
 
-        dd3 = wait_for_element(self.driver, MobileBy.ID, 'com.example.android.apis:id/drag_dot_3')
-        dd2 = self.driver.find_element_by_id('com.example.android.apis:id/drag_dot_2')
+        dd3 = wait_for_element(self.driver, MobileBy.ID, 'io.appium.android.apis:id/drag_dot_3')
```

Comment:
It makes sense to extract the application identifier into a constant, so next time it is replaced it would not be necessary to search for all occurrences of it

Reply:
Thanks, exactly.